pkgs/development/python-modules/capstone/default.nix

```diff
@@ -4,18 +4,23 @@
   capstone,
   stdenv,
   setuptools,
-  pythonAtLeast,
+  fetchpatch,
 }:
 
 buildPythonPackage rec {
   pname = "capstone";
   version = lib.getVersion capstone;
-  format = "setuptools";
-
-  # distutils usage
-  disabled = pythonAtLeast "3.12";
 
   src = capstone.src;
+  patches = [
+    # this patch will be included in the 5.0.2 release
+    (fetchpatch {
+      name = "drop-distutils-in-python-binding.patch";
+      url = "https://github.com/capstone-engine/capstone/commit/a67fa1fed374215734174353418b6a466d522726.patch";
```

Comment:
This commit isn't in an upstream branch - perhaps it was taken from a PR that got rebase-merged? I'm guessing the commit you want is https://github.com/capstone-engine/capstone/commit/d63211e3acb64fceb8b1c4a0d804b4b027f4ef71

Reply:
Yep. I've update the URL to the rebased version of the patch. Thanks!